dev-server/ui-playground/shared/components/PlaygroundApp.js

```diff
@@ -1,10 +1,17 @@
 import { SvgIcon } from '@hypothesis/frontend-shared';
 
-import MenuPatterns from './patterns/MenuPatterns';
 import SharedButtonPatterns from './patterns/SharedButtonPatterns';
 
 import { useRoute } from '../router';
 
+/**
+ * @typedef PlaygroundRoute - Route "handler" that provides a component (function)
+ *   that should be rendered for the indicated route
+ * @prop {RegExp|string} route
```

Comment:
Is this right?

Reply:
Yes. Could do with an example of the format for string routes.